assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.test.js

```diff
@@ -67,6 +72,8 @@ describe( 'ConversionReportingNotificationCTAWidget', () => {
 	);
 
 	beforeEach( () => {
+		enabledFeatures.add( 'conversionReporting' );
```

Comment:
Since this component will surface only when feature flag is enabled, moved the inclusion of feature flag here, instead of it being added in individual tests